aiidalab_widgets_base/databases.py

```diff
@@ -58,6 +59,14 @@ def __init__(self, **kwargs):
         ]
         super(CodQueryWidget, self).__init__(children=children, **kwargs)
 
+    @observe('manager')
+    def _change_manager(self, value):
+        """Set structure manager trait."""
```

Comment:
I think one line of explanation of what this actually does would be helpful here (and in the other places where this
```suggestion
        """Set structure manager trait
        
        Links `input_structure` trait of manager to `structure` trait.
        ."""
```

Reply:
I can do that, thanks for the suggestion.

Reply:
We discussed with @csadorf and agreed to replace the trait with `connect_to_manager` method. There should be much less of the code duplication now.